test/libsolidity/lsp/references/read_write_access_kinds.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import {RGBColor as ThatColor} from "../goto/lib.sol";
+
+contract C
+{
+    function other() public pure returns (ThatColor memory output)
+                                                      //   ^^^^^^ @OutputDef
+    {
+        output.red = 50;
+    //  ^^^^^^ @OutputWrite1
+    //     ^ @OutputWrite
+        output.green = output.red;
+    //  ^^^^^^^^^^^^ @OutputGreenWrite1
+    //  ^^^^^^ @OutputWrite2
+    //                 ^^^^^^ @OutputRead1
+    //         ^ @GreenWrite
+        output.blue = output.green;
+    //                ^^^^^^^^^^^^ @OutputGreenRead1
+    //  ^^^^^^ @OutputWrite3
+    //                ^^^^^^ @OutputRead2
+    //                       ^ @GreenUse
+    }
+}
```

Comment:
Can you add a call to a function with named arguments. The name of the argument should count as a write, right? We should have that covered.